sdk/tests/conformance/context/context-lost-restored.html

```diff
@@ -248,7 +248,8 @@
     // Extension must still be lost.
     var tex = gl.createTexture();
     gl.bindTexture(gl.TEXTURE_2D, tex);
-    shouldGenerateGLError(gl, gl.INVALID_ENUM, "gl.texImage2D(gl.TEXTURE_2D, 0, gl.RGBA, 1, 1, 0, gl.RGBA, gl.FLOAT, null)");
+    wtu.shouldGenerateGLErrorIn(gl, [gl.INVALID_ENUM, gl.INVALID_OPERATION],
```

Comment:
This change looks incorrect. The only error this call is supposed to generate is INVALID_ENUM because of the use of the gl.FLOAT type, since the OES_texture_float extension is lost when the context is restored.